client/components/banner/style.scss

```diff
@@ -8,37 +8,37 @@
 		padding-right: 48px;
 	}
 
-	@include banner-color( $blue-wordpress );
+	@include banner-color( var( --color-primary ) );
 
 	&.is-wpcom-plan {
 		&.is-upgrade-blogger {
-			@include banner-color(darken($alert-green, 20%));
+			@include banner-color( var( --color-success-dark ) );
 		}
 
 		&.is-upgrade-personal {
-			@include banner-color($blue-wordpress);
+			@include banner-color( var( --color-primary ) );
 		}
 
 		&.is-upgrade-premium {
-			@include banner-color($alert-yellow);
+			@include banner-color( var( --color-warning ) );
 		}
 
 		&.is-upgrade-business {
-			@include banner-color($orange-jazzy);
+			@include banner-color( $orange-jazzy );
 		}
 
 		&.is-upgrade-ecommerce {
-			@include banner-color($alert-purple);
+			@include banner-color( $alert-purple );
```

Comment:
@drw158 in general do we have (newly) defined colors for usage in combination with plans?

Reply:
Indeed we do, judging from the new colors on the Plans page. I believe these are the new colors, and afaik we should use these new colors if we see an opportunity. Or maybe not, to prevent possible conflicts when we merge?

Premium: `yellow`
Business: `orange`
eCom: `hot purple`
Blogger: `celadon`
Personal: `hot blue`

Reply:
Oh, it looks like they tried to create approximate colors with the existing colors for plans before we started importing the new studio colors. I'll make some comment above.